### PR TITLE
fix(form-group): allow group classname - TWIG-35

### PR DESCRIPTION
**Pull request by papegaill**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

**Comment by planctus** [comment]
@papegaill, mind that i had to fix conflicts in this branch on the md file and the twig one, please verify that the result is correct.. ;)

File: src/ec/packages/ec-component-text-input/text-input.html.twig [hunk 1]
```diff
@@ -10,6 +10,7 @@
   - "name" (string) (default: '')
   - "type" (string) (default: 'text')
```
Comment (planctus) [inline src/ec/packages/ec-component-text-input/text-input.html.twig]: i noticed that we set the placeholder variable here even if we don't use it anymore... ;)

File: src/ec/packages/ec-component-text-area/text-area.html.twig [hunk 2]
```diff
@@ -10,6 +10,7 @@
   - "hide_label" (boolean) (default: '')
   - "placeholder" (string) (default: '')
```
Comment (planctus) [inline src/ec/packages/ec-component-text-area/text-area.html.twig]: I've checked and i don't see any placeholder in ECL textarea, but here it comes a question: we had tickets for updating form fields, did we touch also the text-area or was it forgotten..?

File: src/ec/packages/ec-component-text-input/text-input.html.twig [hunk 3]
```diff
@@ -10,6 +10,7 @@
...
   - "width" (string) (default: '') 
+  - "extra_group_classes" (optional) (string) (default: '') Extra classes (space separated) for the text-input group
```
Comment (planctus) [inline src/ec/packages/ec-component-text-input/text-input.html.twig]: i think we miss this in the md file for the text-input ;)

File: src/ec/packages/ec-component-text-input/text-input.html.twig (hunk 1 above)
Comment (papegaill) [reply src/ec/packages/ec-component-text-input/text-input.html.twig]: done, I'll update this for the other components too..

File: src/ec/packages/ec-component-text-area/text-area.html.twig (hunk 2 above)
Comment (papegaill) [reply src/ec/packages/ec-component-text-area/text-area.html.twig]: done, checking to update other components too..

File: src/ec/packages/ec-component-text-input/text-input.html.twig (hunk 3 above)
Comment (papegaill) [reply src/ec/packages/ec-component-text-input/text-input.html.twig]: done.

**Pull request merged**